test/test-options-credentials.ts

```diff
@@ -101,7 +101,7 @@ describe('test-options-credentials', function() {
 
   it('should ignore keyFilename if credentials is provided', function(done) {
     const fileCredentials = require('./fixtures/gcloud-credentials.json');
-    const credentials = {
+    const credentials: {[key: string]: string|undefined} = {
```

Comment:
Is this not of type `AuthenticationConfig` from `@google-cloud/common`?

Reply:
Yes, but the compiler expected an index signature, which is why I annotated it this way.